sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -2309,6 +2310,126 @@ class HiveDDLSuite
     }
   }
 
+  test("SPARK-20680: Spark-sql do not support for void column datatype") {
```

Comment:
Could you adjust this test case according to the last commit? Specifically, for the following?
- `void column datatype` -> `unknown column datatype`
- `tabVoidType` -> `tabUnknownType`
- `Forbid CTAS with null type` -> `Forbid CTAS with unknown type`
- `Forbid Replace table AS SELECT with null type` -> `Forbid Replace table AS SELECT with unknown type`